easybuild/easyconfigs/r/rstudio/rstudio-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'rstudio'
```

Comment:
I know history has brought us to this point, but Shouldn't this be called `RStudio-Server`?

Reply:
Yep, and you fixed that in #11764 , thanks!